code/controllers/master.dm

```diff
@@ -80,6 +80,8 @@ GLOBAL_VAR_INIT(CURRENT_TICKLIMIT, TICK_LIMIT_RUNNING)
 
 /datum/controller/master/Shutdown()
 	processing = FALSE
+	sortTim(subsystems, /proc/cmp_subsystem_init)
+	subsystems = reverselist(subsystems)
```

Comment:
use `reverserange()` instead and just remove the commented out proc from lists.dm